engines/docker/engine.go

```diff
@@ -100,6 +121,13 @@ func (e *engine) NewSandboxBuilder(options engines.SandboxOptions) (engines.Sand
 	var p payloadType
 	schematypes.MustValidateAndMap(e.PayloadSchema(), options.Payload, &p)
 
+	p.Devices = funk.UniqString(p.Devices)
```

Comment:
Is this for deduplication? If so, just add `Unique: true` in the JSON schema for `array`...

Reply:
In a follow up when we have more options.